README-Gitpod.md

```diff
@@ -0,0 +1,39 @@
+Token Contract in AssemblyScript - Gitpod version
+=================================================
+
+This is a stripped-down example where we'll demonstrate creating a token. We'll visit a page, sign in and use your browser's console to run a commands to initialize, send, and get the balance of a custom token.
+
+This README is specific to Gitpod and this example. For local development, please see [README.md](README.md).
+
+## Description
+
+This project contains implementation of token contract similar to [ERC20](https://theethereum.wiki/w/index.php/ERC20_Token_Standard).
+
+## Getting started
+
+Gitpod has taken care of installing all the necessary tools and dependencies. At the bottom of Gitpod is a terminal which will display a link to follow:
+```bash
+Server running at http://localhost:1234
+```
+
+A small dialog may appear showing options similar to this:
+
+![Gitpod dialog that appears when website is served](assets/gitpod-port-1234.jpg)
+
+The "Preview" option will open the site in a tab within the IDE. Note that Gitpod may need a little time to spin up the website. It's possible this step might require reloading after a brief pause.
+
+Once you've opened the tab in your browser, follow the directions displayed on the web page by copy and pasting the commands into the browser console.
+
+In many modern browsers you may find this by right clicking anywhere on the page, right-click, **Inspect**, and then navigate to the **Console** tab.
+
+## To Test
+
+```
+yarn asp // as-pect tests
+yarn jest // integration tests
+yarn test // both
+```
```

Comment:
Rather than a single code block for these, I'd rather we used bullet points with some code in them. That gives us a place to explain & link to `asp` and `jest`, which I think is important since it's really hard to figure out where the `asp` command comes from without having a link to it. 

Alternatively, rather than have a whole heading for testing, we could include a sentence about testing similar to [the one in Guest Book](https://github.com/near-examples/guest-book/blob/7d65ffb4b9e2223681dcbe4fe55d005242aaaf27/README.md#exploring-the-code)

Reply:
did it like guestbook

Reply:
I don't see that: https://github.com/near-examples/token-contract-as/pull/10/files#diff-7e401b000c7620d70d4f73ef1e11f90aR31-R36

Do you have a local commit you need to push? I'm happy to update it myself, I just want to make sure.